2022/questions.md

```diff
@@ -0,0 +1,531 @@
+## Open Source Program Office (OSPO) 2022 Survey
+The TODO Group, together with Linux Foundation Research and The New Stack, is conducting a survey as part of a research project on the prevalence and outcomes of open source programs among enterprises across the globe. 
+
+Personally identifiable information will not be used without your explicit permission. The information collected in this survey will be used as part of a research study. 
+Only one response per company will be included in the study's results. 
+
+*This survey should take no more than 15 minutes of your time.*
+
+
+PRIVACY
+
+Your name and company name will not be published. Reviews are attributed to your role, company size, and industry. Responses will be subject to the Linux Foundation's Privacy Policy, available at https://linuxfoundation.org/privacy. Please note that survey partners who are not Linux Foundation employees will be involved in reviewing the survey results. Although survey partners are not permitted to use your personal data for other purposes, if you would prefer that your name or email address would not be visible in the results that the survey partners review, then please do not include them in your responses.
+
+
+VISIBILITY
+
+We will summarize the survey data and share the findings during OSPOCon 2022. The summary report will be published on both the TODO Group and Linux Foundation websites. 
+
+QUESTIONS
+
+If you have questions regarding this survey, please email us at research@linuxfoundation.org.
+
+
+## Part 1
+1. Where is your company or organization on its open source journey? (Check all that apply) 
+* Consuming open source code in products or services
+* Contributing to upstream open source projects
+* Influencing open source projects via leadership or maintainer roles
+* Initiating or releasing open source projects
+* Collaborating with peers across open source projects and/or foundations
+* Not involved in open source
+* Don’t know
```

Comment:
```suggestion
## Part 1

1. Where is your company or organization on its open source journey? (Check all that apply) 
    * Consuming open source code in products or services
    * Contributing to upstream open source projects
    * Influencing open source projects via leadership or maintainer roles
    * Initiating or releasing open source projects
    * Collaborating with peers across open source projects and/or foundations
    * Not involved in open source
    * Don’t know
```

helpful documentation fairy tip: if you don't format your lists like this sometimes you'll run into issues with formatting displaying incorrectly in GitHub pages <3

- Always a full line (carriage return) after any header and before any text directly below it
- tab in for any subitems of a list to properly display them as a sublist 
   - for example, this is tabbed in to the bullet point above

Reply:
You are totally right @celestehorgan thanks for making this format review!